src/ui/resources.qrc

```diff
@@ -1,23 +1,35 @@
 <RCC>
     <qresource prefix="/ui">
         <file>resources/android-active.svg</file>
+        <file>resources/android-active-dark.svg</file>
         <file>resources/android.svg</file>
         <file>resources/animations/globe_animation.json</file>
         <file>resources/animations/lock_animation.json</file>
         <file>resources/animations/vpnactive_animation.json</file>
         <file>resources/animations/vpnlogo-drop_animation.json</file>
         <file>resources/apple-active.svg</file>
+        <file>resources/apple-active-dark.svg</file>
         <file>resources/apple.svg</file>
         <file>resources/check-email.svg</file>
+        <file>resources/check-email-dark.svg</file>
         <file>resources/connection-info.svg</file>
         <file>resources/connection-info-dark.svg</file>
+        <file>resources/connection-info-smaller-dark.svg</file>
         <file>resources/copy.svg</file>
+        <file>resources/copy-dark.svg</file>
         <file>resources/create-password.svg</file>
         <file>resources/data-collection.svg</file>
         <file>resources/devices.svg</file>
+        <file>resources/devices-dark.svg</file>
         <file>resources/devicesLimit.svg</file>
+        <file>resources/DevicesLimit - dark.svg</file>
```

Comment:
lowercase and spaces :)

Reply:
I've updated all assets with spaces or a leading capital letter.